.github/workflows/wix.yml

```diff
@@ -25,7 +25,6 @@ jobs:
           SUB_TAG: 8u362b09
           JVM: hotspot
         - jdk: 11
-          ICEDTEA_WEB_VERSION: ""
```

Comment:
Would we not set this to 1.8.8?

Reply:
ITW only works with Java 8 so we skip on 11 and 17